VmAgent.Core/Interfaces/ProcessRunner.cs

```diff
@@ -76,6 +79,27 @@ public override Task CollectLogs(string processId, string logsFolder, ISessionHo
             // The game server is free to read the env variable for log folder and write all output to a file in that folder.
             // If required, we can add action handlers (see SystemOperations.RunProcess for example).
             // However, keeping the file handle around can be tricky.
+            throw new NotSupportedException();
+        }
+
+        public override Task CreateExceptionLogs(string logsFolder, string exceptionMessage)
+        {
+            try
+            {
+                _logger.LogVerbose("Collecting logs for failed start game process.");
+                string destinationFileName = Path.Combine(logsFolder, ConsoleLogCaptureFileName);
+
+                using (StreamWriter sw = File.AppendText(destinationFileName))
+                {
+                    sw.WriteLine(DateTime.Now.ToString("HH:mm:ss D"));
+                    sw.WriteLine($"{exceptionMessage}");
+                }
+                _logger.LogVerbose($"Written logs for failed start game process to {destinationFileName}.");
+            }
+            catch (Exception ex)
+            {
+                _logger.LogException($"Failed to write failed start game error logs for Windows process", ex);
```

Comment:
this method will be called on Linux process game servers as well (since there is no "if" for it), maybe remove the "Windows" from the logging message?